content/docs/sidebar.json

```
@@ -134,6 +134,12 @@
           "merge-conflicts"
         ]
       },
+      {
+        "slug": "checkpoints",
+        "tutorials": {
+          "katacoda": "https://katacoda.com/dvc/courses/get-started/checkpoints"
+        }
+      },
       "experiment-management",
```

Let's put this right after the main Exps guide? Committable suggestion:
```suggestion
      "experiment-management",
      {
        "slug": "checkpoints",
        "tutorials": {
          "katacoda": "https://katacoda.com/dvc/courses/get-started/checkpoints"
        }
      },
```